setup.py

```diff
@@ -23,13 +23,17 @@
         'tifffile',
         'torch==1.0',
         'tqdm',
+        'scikit-image==0.15.0',
     ],
     extras_require={
         'dev': [
             'flake8',
             'pylint',
             'pytest',
             'pytest-cov',
+        ],
+        'examples': [
+            't4==0.0.10',
```

Comment:
I don't know much about `t4`, but please double check if this needs to be pinned.

Reply:
It's api is in development, so I think pinning it is a safe thing to do.